### ISSUE TITLE
Enable JWT for REST API

### ISSUE THREAD

**Issue by zaibon** [zero-os/0-orchestrator#449]
The orcestrator API has to be protected by JWT. 
The orchestrator is deployed for a specific IYO organization and only member of this organization can send request to the API.

Raml has to be updated with the `securedBy` node.
example : 
```raml
securedBy: [itsyouonline: { scopes: [ "user:memberof:organization_name" ] }]
```

Then the route interfaces has to be update to required the scope (https://github.com/zero-os/0-orchestrator/blob/master/api/node/node_if.go#L324)

you can generate the server in a temporary folder to see how go-raml creates it.

**Comment by ahmedelsayed-93** [comment]
verification blocked on https://github.com/zero-os/0-orchestrator/issues/481

**Comment by ahmedelsayed-93** [comment]
the duration of JWT token is 1 day, so if the owner of the organization remove user X form the organization,  user X will still be able to access the apis until the JWT token expire.
so we need to reduce the duration of the JWT token

**Comment by grimpy** [comment]
@ahmedelsayed-93 altough a valid concern, this does not task wasnt properly implement this, please validate this ticket for the feature it implemnts, and create a new issue for your findings
